VideoLocker/res/layout/row_course_list.xml

```diff
@@ -8,67 +8,186 @@
+            android:layout_height="wrap_content"
+            android:background="@drawable/course_facepile_backing"
+            android:orientation="vertical"
+            android:layout_above="@+id/course_item_detail"
+            android:paddingTop="22dp"
+            android:paddingRight="8dp"
+            android:paddingLeft="8dp"
+            android:paddingBottom="8dp"
+            android:gravity="bottom">
+
+            <org.edx.mobile.view.custom.ETextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="2dp"
+                style="@style/semibold_text"
+                android:textColor="@color/white"
+                android:textSize="12sp"
+                android:shadowColor="@color/black"
+                android:shadowDx="0"
+                android:shadowDy="1"
+                android:shadowRadius="2"
+                android:text="@string/course_friends_also_enrolled"/>
+
+            <org.edx.mobile.view.custom.SocialFacePileView
+                android:id="@+id/course_item_facepileview"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:avatarGap="8dp"
+                app:overflowTextSize="20dp"
+                app:borderRadius="@dimen/avatar_image_radius"
+                app:avatarCount="6"
+                app:avatarMaxSize="50dp"
+                android:gravity="center|left|start"
+                tools:targetApi="17" />
+
+        </LinearLayout>
 
         <RelativeLayout
-            android:id="@+id/course_item_top_display"
+            android:id="@+id/course_item_detail"
             android:layout_width="match_parent"
-            android:layout_height="120dp">
+            android:layout_height="@dimen/course_detail_card_height"
+            android:layout_alignParentBottom="true"
+            android:background="@color/transparent_white_85" >
 
-            <com.android.volley.toolbox.NetworkImageView
-                android:id="@+id/course_image"
-                tools:background="@drawable/sample_course_image"
+            <org.edx.mobile.view.custom.ETextView
+                android:id="@+id/course_name"
+                style="@style/bold_grey_text"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:layout_alignParentTop="true"
-                android:adjustViewBounds="true"
-                android:scaleType="centerCrop" />
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="8dp"
+                android:layout_marginLeft="10dp"
+                android:layout_marginRight="10dp"
+                android:layout_marginTop="10dp"
+                android:text="@string/course_name"
+                android:textSize="16sp" />
 
             <LinearLayout
```

Comment:
This `LinearLayout` seems to be useless, as it has only one child, and all it's attributes could be transferred to it.